h/buildext.py

```diff
@@ -93,21 +117,14 @@ def chrome_manifest(request):
         version = h.__version__
         version_name = 'Official Build'
 
-    src = request.resource_url(request.context)
-
-    # We need to use only the host and port for the CSP script-src when
```

Comment:
Actually this is preserving the previous behavior, the "script_host_url" passed as the context when rendering the manifest is the host + port.

What has changed is that now we always include just the origin of the URL serving the assets, never the path. Previously if you tried to run your local H instance on a host other than localhost, the CSP entry in the manifest would end up including the path as well.